codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -262,29 +262,21 @@ private int maxUsableChannelBytes() {
     }
 
     /**
-     * Package private for testing purposes only!
+     * This will allocate bytes by stream weight and priority for the entire tree rooted at {@code parent}, but does not
```

Comment:
Not sure this comment applies anymore since the priority tree algorithm was moved to another class.